cwms-data-api/src/main/java/cwms/cda/formatters/json/JsonV1.java

```diff
@@ -26,145 +21,153 @@
 import cwms.cda.formatters.OfficeFormatV1;
 import cwms.cda.formatters.OutputFormatter;
 import io.javalin.http.BadRequestResponse;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
 import org.jetbrains.annotations.NotNull;
 import service.annotations.FormatService;
 
 @FormatService(contentType = Formats.JSON,
-			   dataTypes = {
-				   Office.class,
-				   Location.class,
-				   Catalog.class,
-				   LocationGroup.class,
-				   LocationCategory.class,
-				   TimeSeriesCategory.class, TimeSeriesGroup.class,
-				   Clob.class,
-				   Clobs.class,
-				   TimeSeriesGroup.class,
-				   RecentValue.class
-				})
+        dataTypes = {
```

Comment:
Review https://github.com/USACE/cwms-data-api/pull/509 again, it cleared after a made a bunch of changes.
(one of us will have to rebase, I don't care which)